pandas/core/groupby.py

```diff
@@ -1476,8 +1476,93 @@ def describe(self, **kwargs):
     @Appender(_doc_template)
     def resample(self, rule, *args, **kwargs):
         """
-        Provide resampling when using a TimeGrouper
-        Return a new grouper with our resampler appended
+        Provide resampling when using a TimeGrouper.
+
+        Given a grouper the function resamples it according to a string and an
+        optional list and dictionary of parameters. Returns a new grouper with
```

Comment:
"optional list and dictionary of parameters" doesn't mean much to me as a user. I'd say remove that bit.